Source/MQTTnet.Extensions.ManagedClient/IManagedMqttClient.cs

```diff
@@ -16,36 +16,95 @@ public interface IManagedMqttClient : IApplicationMessageReceiver, IApplicationM
         /// </summary>
         IMqttClient InternalClient { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether the client is started or not.
```

Comment:
For me this comment has no valuable information because it is only the long version of the property name.